ingeniamotion/pdo.py

```diff
@@ -301,6 +299,34 @@ def high_precision_sleep(duration: float) -> None:
             while duration - (time.perf_counter() - start_time) > 0:
                 pass
 
+        def __set_watchdog_timeout(self) -> None:
+            if self._watchdog_timeout is None:
+                self._watchdog_timeout = max(
+                    self.DEFAULT_WATCHDOG_TIMEOUT,
+                    self._refresh_rate * self.PDO_WATCHDOG_INCREMENT_FACTOR,
+                )
+                is_watchdog_timeout_manually_set = False
+            else:
+                is_watchdog_timeout_manually_set = True
+            try:
+                for servo in self._net.servos:
+                    servo.set_pdo_watchdog_time(self._watchdog_timeout)
+            except AttributeError as e:
+                max_pdo_watchdog = re.findall(r"\d+\.\d+|\d+", e.__str__())
```

Comment:
This looks sketchy. I'm not sure if the formatting of this line can be considered a stable API: https://github.com/bnjmnp/pysoem/blob/1b730b6d97e95d1e7bde5bb0de471fe11d85119d/src/pysoem/pysoem.pyx#L1073

What if we contributed with a pysoem function that was get_watchdog_maximum_limit()

Reply:
Done https://github.com/bnjmnp/pysoem/pull/169